File: FizzBuzz/FizzBuzz.js
var lowerLimit = 1
var upperLimit = 100

for (var i = lowerLimit; i <= upperLimit; i++)
{
	if (i % 3 == 5 && i % 5 == 0)
	{
		console.log("FizzBuzz")	
	}

	else if (i % 3 == 0)
	{
		console.log("Fizz")	
	}

	else if (i % 5 == 0)
	{
		console.log("Buzz")	
	}

	else
	{
		console.log(i)
	}
}

File: FizzBuzz/FizzBuzz.py
lowerLimit = 1
upperLimit = 100

for i in range(lowerLimit, upperLimit + 1):
	if (i % 3 == 0 and i % 5 == 0):
		print("FizzBuzz")
	
	elif (i % 3 == 0):
		print("Fizz")

	elif (i % 5 == 0):
		print("Buzz")

	else:
		print(i)